Docs/Whats_New.md

```diff
@@ -23,7 +23,10 @@ Released TBD
 - Fix potential crash when using multi-planar images.
 - Ensure buffers available for buffer addresses in push constants.
 - Support `libMoltenVK.dylib` for _iOS Simulator_ architecture.
-- Restore support for _iOS Simulator_ destination in recent update to _Cube_ demo that uses dynamic-linking.
+- Restore support for _iOS Simulat
+Speor_ destination in recent update to _Cube_ demo that uses dynamic-linking.
```

Comment:
```suggestion
- Restore support for _iOS Simulator_ destination in recent update to _Cube_ demo that uses dynamic-linking.
```

I take it you didn't mean to insert those spurious characters there.

Reply:
WTF? I must have sneezed with my hands over the keyboard!

Thanks. You're proofreading is, as always, impeccable!